projects/Benchmarks/Benchmarks.csproj

```diff
@@ -1,21 +1,20 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <AssemblyOriginatorKeyFile>../rabbit.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
-    <TargetFrameworks>netcoreapp3.1;net48</TargetFrameworks>
-    <ReleaseVersion>8.0</ReleaseVersion>
+    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
```

Comment:
I'm confused now. I thought you are going to dual target net6 and net472 based on the previous comment?